checkpointing/tensorflow/README.md

```diff
@@ -0,0 +1,23 @@
+## Checkpointing in Tensorflow with conda environment and GPU support
+The example uses a pre-configured conda environment to demonstrate how one might implement model checkpointing in Tensorflow.
+
+The Tensorflow example downloads the MNIST dataset and trains a neural net on it for 20 epochs, making a "checkpoint" every fifth epoch. In the event of a runtime interruption, these checkpoints are preserved on the submit server and are used to resume training, preventing the loss of already-completed work.
+
+Checkpointing can be useful when running jobs that take multiple days to train or when training on resources that may be interrupted, like OSG GPUs.
+
+The five files include:
+- `checkpoint.sub` ## used to submit the job on HTCondor.
+- `run.sh` ## the executable called by `checkpoint.sub`.
+- `tf_checkpointing.py` ## the python program with an a checkpointing implementation.
+- `tf_checkpointing.tar.gz` ## the conda environment used by HTCondor for tensorflow dependencies -- this is hosted on a squid web server and is not included in the repo.
+- `environment.yml` ## The environment file used to create the conda env used in this example. This file is not used by the job. The environment is included in `tf_checkpointing.tar.gz`.
+
+### Usage
+- Log into the HTC system.
+- Clone this repository: `git clone https://github.com/CHTC/template-GPUs`.
+- `cd` into this folder: `cd template-GPUs/checkpointing/tensorflow`
+- Because conda environments tend to be large, a Squid caching server is used to host the environment. You can access the environment `.tar.gz` file from the submit node at `/squid/gpu-examples`. For more information about using Squid, please review the CHTC guide:
```

Comment:
```suggestion
- Because conda environments tend to be large, a Squid caching server is used to host the environment. You can view the environment `.tar.gz` file from the submit node at `/squid/gpu-examples`. For more information about using Squid, please review the CHTC guide:
```